allauth/account/models.py

```diff
@@ -41,6 +43,7 @@ class Meta:
                     condition=Q(verified=True),
                 )
             ]
+        indexes = [Index(Upper("email"), name="idx_upper_email")]
```

Comment:
The other indexes are named like this:

```
Indexes:
    "account_emailaddress_pkey" PRIMARY KEY, btree (id)
```

Could you rename this one to `account_emailaddress_upper_email` ?

Reply:
I have renamed the index and manually renamed in migration file, i hope that's the correct way instead of deleting and regenerating the migration file.